docs/.vitepress/theme/components/Home.vue

```diff
@@ -125,7 +125,7 @@ const subscribeToNewsletter = async () => {
           :disabled="isBusy"
           id="email"
           type="email"
-          pattern="^[a-zA-Z0-9]+(?:[._+-][a-zA-Z0-9]+)*@[a-zA-Z0-9-]+\.[a-zA-Z]{2,}$"
+          pattern="^[a-zA-Z0-9]+([._+\-][a-zA-Z0-9]+)*@[a-zA-Z0-9\-]+\.[a-zA-Z]{2,}$"
```

Comment:
I don't think you need to escape the `-` -- if it's at the end of the range it's treated as a character (at least in regex parsers I'm familiar with)

Reply:
thats what was my thinking too, but if i dont i get an `invalid range` error :D

Reply:
check how i escaped both dashes, in the user and in the domain. #fun